.mergify.yml

```
@@ -0,0 +1,11 @@
+queue_rules:
+ - name: default
+   conditions: []
+
+pull_request_rules:
+  - name: Automatic merge on approval
```

AFAIK it is not an automatic merge, it just puts the PR in the merge queue once `conditions` are met.

```suggestion
  - name: Use merge queue
```